src/pages/Build.vue

```diff
@@ -787,7 +783,7 @@
       testingCompleted () {
         let testing_completed = true
         for (let task of this.build.tasks) {
-          if (task.status < BuildStatus.TEST_COMPLETED) {
+          if (task.status <= BuildStatus.COMPLETED || task.status == BuildStatus.TEST_CREATED || task.status == BuildStatus.TEST_STARTED) {
```

Comment:
You have changed logic of `testingCompleted` but name stayed the same
Maybe we should rename it to `testingInProgessOrCompleted`?

or instead of changing `testingCompleted` add new computed item `testingInProgress`?
it`ll be easier to understand in the future IMO

Reply:
No, I didn't change the logic of this function. This condition will return `true` only for not completed tasks.

Reply:
I might be mistaken but there is change of logic
**before:** 
testingCompleted will return true with TEST_COMPLETED, TEST_FAILED, ALL_TESTS_FAILED -> i.e. package was build and tested (no matter if tests were failed or not)

**after**:
  testingCompleted will return true if FAILED, EXCLUDED, CANCELLED, TEST_COMPLETED, TEST_FAILED, ALL_TESTS_FAILED,
  i.e.  build failed, excluded, cancelled **or** package was build and tested (no matter if tests were failed or not) .

Maybe it\`ll be more cleaner create new compute  buildCompleted that will return false if FAILED, EXCLUDED, CANCELLED and combine two computes like this
`  
 v-if="!buildCompleted | testingCompleted"?
`

  But if everyone else disagree with me, I`ll resolve this dialog

Reply:
We already have `buildFinished()` and I don't understand why we should create some new compute

Reply:
Just thought that returning testFinished=true for packages that were not even built is a bit counterintuitive.
But if you dont see problem here, I`ll approve PR